lib/resources/example_json/r4_examples_patient.rb

```diff
@@ -67,6 +67,76 @@ module Resources
                '<p><b>Marital Status</b>: Married</p></div>'
       },
       'extension': [
+        {
+          'id': '59434424',
+          'extension': [
+            {
+              'valueCodeableConcept': {
+                'coding': [
+                  {
+                    'system': 'https://fhir.cerner.com/eb2384f8-839e-4c6e-8b29-18e71db1a0b1/codeSet/4640016',
```

Comment:
Update the tenant id in all examples to "ec2458f2-1e24-41c8-b71b-0e701af7583d" which is the public sandbox. Ideally the examples that we give here should be reproducible. For now update the tenant id

Reply:
updated